### PR TITLE
Added Profile page

### PR DESCRIPTION
**Pull request by lucasungaretti**
made the profile button functional and added the page where profile will be

### CODE REVIEW

File: travvy/src/components/Home.vue [hunk 1]
```diff
@@ -5,7 +5,7 @@
   <div class="Home">
     <div class="container">
     <!-- Profile button that will lead to the users profile page -->
-    <input type="button" value="Profile" v-on:click="reverseMessage" style="float: right;" class="profile">
+    <router-link to="/Profile" tag="button" class="button" style="float: right;"><span>Profile</span></router-link>
```
Comment (lkdfount) [inline travvy/src/components/Home.vue:8]: We've had a few problems with router-link, might be a better idea to use a button that takes you to profile

**Comment by emilynairnn** [comment]
Looks good - excited to see how we can use the profile page to customize the website more!

File: travvy/src/components/Home.vue (hunk 1 above)
Comment (lucasungaretti) [reply travvy/src/components/Home.vue:8]: that was my original plan but I saw the use of router-link for some other buttons and thats why I used it

File: travvy/src/components/Profile.vue [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<template>  
+  <div class="Profile">
+        <h2>Profile</h2>
+        <br>
+        <br>
```
Comment (lkdfount) [inline travvy/src/components/Profile.vue:5]: This is good. Maybe soon we can try to display the name to make sure the user's email from the store.

**Pull request merged**